samples/apps/spring-petclinic/Makefile

```diff
@@ -0,0 +1,72 @@
+PETCLINIC_COMMIT ?= ce626da705302e71771b7e22429d1cc3209e7ebb
+
+PETCLINIC_REPO ?= quay.io/pmacik/pet-clinic
+PETCLINIC_APP_IMAGE ?= $(PETCLINIC_REPO):app
+PETCLINIC_APP_DBINIT_IMAGE ?= $(PETCLINIC_REPO):dbinit
+NAMESPACE ?= my-postgresql
+
+# one of "openshift", "minikube"
+KUBERNETES_RUNTIME ?= minikube
+
+.PHONY: namespace
+namespace:
+ifeq ($(KUBERNETES_RUNTIME), openshift)
+	-oc new-project $(NAMESPACE)
+else ifeq ($(KUBERNETES_RUNTIME), minikube)
+	-kubectl create namespace $(NAMESPACE)
+	kubectl config set-context --current --namespace=$(NAMESPACE)
+endif
+
+.PHONY: install-sbo
+install-sbo:
```

Comment:
do we really need to have here a target where we install SBO? If needed, we can refer directly `install.sh` from the repo itself, and not download it from `master` branch.

Reply:
this makefile target is for convenience, no need to have it here, thx